extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -94,50 +369,57 @@ class AmpJWPlayer extends AMP.BaseElement {
 
     this.contentSearch_ = element.getAttribute('data-content-search') || '';
     this.contentBackfill_ = element.getAttribute('data-content-backfill') || '';
+    this.contentContextual_ =
```

Comment:
This seems like a new attribute. If so, please document on the `.md` file.

Reply:
Good call. This isn't a necessary attribute as it can be inferred from pre-existing attributes, so I removed it.